crates/turbopack-core/src/resolve/mod.rs

```diff
@@ -2553,7 +2619,18 @@ pub async fn handle_resolve_error(
                 .cell()
                 .emit();
             }
-            result
+
+            if let Some(error_string) = result.await?.get_first_error() {
```

Comment:
I think it should not do that. It should just return the result even if that's a resolve result with runtime error.

The issue should be emitted earlier...